starter/src/main/java/org/teiid/spring/autoconfigure/TeiidServer.java

```diff
@@ -272,6 +288,10 @@ void deployVDB(VDBMetaData vdb, boolean last) {
                         }
                     }
                 }
+            } else {
+                // embedded engine does not allow vdb-scoped transaltors, however these are
+                // already handled during the load in spring boot.
+                vdb.getOverrideTranslatorsMap().clear();
```

Comment:
If you want this information associated with the vdb, then you don't have to clear this here.  Just override EmbeddedServer.allowOverrideTranslators

Reply:
Ok, I will fix it